bin/gxa-index-set-no-autocreate.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+SCHEMA_VERSION=${SCHEMA_VERSION:-1}
```

Comment:
```suggestion
SCHEMA_VERSION=1
```
lets hard code version numbers as in the other scripts.